assets/javascripts/openqa.js

```diff
@@ -290,28 +290,45 @@ function renderSearchResults(query, url) {
     }
     spinner.style.display = 'none';
     var heading = document.getElementById('results-heading');
-    heading.appendChild(document.createTextNode(': ' + json.data.length + ' matches found'));
+    heading.appendChild(document.createTextNode(': ' + json.data.total_count + ' matches found'));
     var results = document.createElement('div');
     results.id = 'results';
     results.className = 'list-group';
-    json.data.forEach(function (value, index) {
-      var item = document.createElement('div');
-      item.className = 'list-group-item';
-      var header = document.createElement('div');
-      header.className = 'd-flex w-100 justify-content-between';
-      var title = document.createElement('h5');
-      title.className = 'occurrence mb-1';
-      title.appendChild(document.createTextNode(value.occurrence));
-      header.appendChild(title);
-      item.appendChild(header);
-      if (value.contents) {
-        var contents = document.createElement('pre');
-        contents.className = 'contents mb-1';
-        contents.appendChild(document.createTextNode(value.contents));
-        item.appendChild(contents);
+    const types = ['code', 'modules', 'templates'];
+
+    for (var i = 0; i < types.length; i++) {
```

Comment:
And this `var` should be a `let`.

Reply:
I changed it into an iterator

Reply:
I find loops cleaner looking personally. Arrow functions are now also preferable to use whenever possible, instead of the `keyword` function.

Reply:
I changed the variable into an object and didn't know how to do it in a for loop :)